index.html

```diff
@@ -26,12 +28,14 @@
     </div>
   </section>
   <section class="sec-contact" id="sec-contact">
-    <ul class="soc-media">
-      <li><a href="https://www.facebook.com" class="fa fa-facebook" target="_blank"></a></li> // add facebook link
-      <li><a href="https://www.twitter.com" class="fa fa-envelope"></a></li>  // add twitter link
-    </ul>
-  </section>
+   <ul class="soc-media">
+     <li><a href="https://mobile.twitter.com/gurpsohal/tweets" class="fa fa-twitter" target="_blank"></a></li>
+
+   </ul>
+ </section>
   <section class="sec-home" id="sec-home">
+    <img class="picture"src="/Users/gpreet/Documents/IMG_4078.JPG"/>
```

Comment:
This is using the path on your computer, not the path for the webpage. You need to put the image into your project folder and give the relative path from index.html to it, just like your script or JS files.